hugolib/page.go

```diff
@@ -380,6 +380,17 @@ var (
 	internalSummaryDivider = []byte("HUGOMORE42")
 )
 
+// If the raw content has nothing but whitespace after the summary marker then
+// the page shouldn't be marked as truncated.  This check is simplest against
+// the raw content because different markup engines (rst and asciidoc in
+// particular) add div and p elements after the summary marker.
+func (p *Page) anyContentAfterUserDefinedSummary() bool {
+	sections := bytes.Split(p.rawContent, helpers.SummaryDivider)
```

Comment:
I care about performance, and I'm pretty sure we have done the above split before. So no need to do it again.